test/provider_os_memory_multiple_numa_nodes.cpp

```diff
@@ -28,7 +30,17 @@ std::vector<int> get_available_numa_nodes_numbers() {
     return available_numa_nodes_numbers;
```

Comment:
This test is all about multiple numa nodes. I think there should be a check if the size of available numa nodes vector is greater than 1. Otherwise, we might think that we test umf on multiple numa nodes but actually testing just a single numa node.